packages/cozy-pouch-link/src/platformWeb.js

```diff
@@ -18,6 +18,9 @@ const storage = {
   },
   removeItem: async key => {
     return window.localStorage.removeItem(key)
+  },
+  destroy: async () => {
+    return window.localStorage.clear()
```

Comment:
The side effect of clearing the entire localStorage may be problematic as this is not the only mechanism relying on the local storage.

For example, on the Flagship app we expect to call the `link.reset()` method if we need to purge the offline data. For now we implemented it through an UniversalLink in this PR: https://github.com/cozy/cozy-flagship-app/pull/1247

With your implementation, the UL would also clear the persisted cozy-client.

I would prefer to call `removeItem()` for each [pouch related key](https://github.com/cozy/cozy-client/blob/800005e88f1b2e63ecaa7deeb81e16d7d8350f11/packages/cozy-pouch-link/src/localStorage.js#L1-L8)

Reply:
> I would prefer to call removeItem() for each [pouch related key](https://github.com/cozy/cozy-client/blob/800005e88f1b2e63ecaa7deeb81e16d7d8350f11/packages/cozy-pouch-link/src/localStorage.js#L1-L8)

That's  what I wanted to avoid, but fair enough, this is less risky

Reply:
For you @Ldoppea [d45e99f](https://github.com/cozy/cozy-client/pull/1552/commits/d45e99ff7cb458896e28d089a2818fcd7ab4447f)